user_guide_src/source/cli/cli_library/001.php

```diff
@@ -2,9 +2,9 @@
 
 namespace App\Controllers;
 
-use CodeIgniter\CLI\CLI;
+use CodeIgniter\Controller;
```

Comment:
This is how to use CLI library.

```suggestion
use CodeIgniter\CLI\CLI;
use CodeIgniter\Controller;
```

Reply:
Thanks